draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3396,6 +3478,9 @@ This section gives Sage {{SAGE}} scripts used to generate parameters for the map
 The below function outputs an appropriate Z for the Shallue and van de Woestijne map ({{svdw}}).
 
 ~~~sage
+# Arguments:
+# - F, a field object, e.g., F = GF(2^521 - 1)
+# - A and B, the coefficients of the curve equation y^2 = x^3 + A * x + B
```

Comment:
@kwantam: Could you please remind me why is required that `g(x)-Z` be irreducible in `find_z_sswu`?

Reply:
Sure thing! It's an attempt to avoid infringing on an existing patent. See [my post](https://mailarchive.ietf.org/arch/msg/cfrg/jV4Wr4fbMKkd4vzsbEhKbous16Y) from the cfrg mailing list.